config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,17 @@
+puppet_class = 'foreman_proxy::plugin::pulp'
+param = 'pulpcore_enabled'
+
+def upgrade
+  answers[puppet_class][param] = true
+  return unless answers['katello']
+  answers['katello'] = {} unless answers['katello'].is_a?(Hash)
+  answers['katello']['use_pulp_2_for_file'] = true
+  answers['katello']['use_pulp_2_for_docker'] = true
+end
```

Comment:
We also need to set https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/init.pp#L131 to false so that on upgrade `/pulp/isos` is being served from Pulp 2

Reply:
Done, thanks